app/models/job.ts

```diff
@@ -576,7 +576,7 @@ export class Job extends DBRecord implements JobRecord {
     currentPage = 0,
     perPage = env.defaultResultPageSize,
   ): Promise<{ job: Job; pagination: ILengthAwarePagination }> {
-    const result = await transaction('jobs').select().where({ username, requestId }).forUpdate();
+    const result = await transaction('jobs').select().where({ username, requestId });
```

Comment:
The two changes in this file were places where we did not need to lock the jobs table as far as I could tell - they were only used for read only things like displaying job status.